### ISSUE TITLE
Stretched video wallpaper

### ISSUE THREAD

**Issue by nas00** [Komorebi-Fork/komorebi#29]
Video wallpaper is stretched vertically and horizontally to fill the screen. Can it be configured to stretch in one direction only or just centre the video instead?

**Comment by Kangie** [comment]
After looking at the code, it doesn't seem like there's a method indicated - video is set to the height and width of the screen Komorebi is on.

A short-term solution would be to re-encode the video at a resolution that matches your screen and add letterboxes (etc) - basically scaling the source video instead.

**Comment by nas00** [comment]
It's a low power arm cpu and no hardware acceleration - not ideal I know. Halving the video resolution, halves the processor usage but the stretch-to-fit behaviour is a bit blurry and choppy on playback compared to native screen resolution.

**Comment by Ev1lbl0w** [comment]
I agree that should be an option on the wallpaper creator. It doesn't seem hard to implement: *Center* just centers the video stream to the center without changing video size, and *Scale* only "stretches" the video without deforming it, creating letterboxing.

**Comment by nas00** [comment]
Perhaps copy the position settings available for the clock/date? I had assumed video wallpapers were comprised of a mostly static background image plus a small video overlayed on top - to save on cpu cycles.

**Comment by mpolidori** [comment]
Any updates on this? If not, does anyone know where in the code to start looking at to implement/fix this?
